File: Python/Activities/Project/Activity_01.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.support.wait import WebDriverWait
import time
service = FirefoxService(GeckoDriverManager().install())

# start the driver
with webdriver.Firefox(service=service) as driver:
    # Navigate to the URL
    driver.get("http://alchemy.hguy.co/orangehrm")
    actualTitle = driver.title
    # assert the title
    assert driver.title, "OrangeHRM"
    print("The Title of the Page is: ", actualTitle)
    driver.close()

File: Selenium/Project/FST_HRM_Project/src/test/java/Activity/Activity_01.java
package Activity;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;

public class Activity_01 {
    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/orangehrm");
        System.out.println(driver.getTitle());
        String Expected_Website=driver.getTitle();

        Assert.assertEquals(Expected_Website,"OrangeHRM");
        driver.close();
    }
}
